packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -17,6 +17,11 @@ export async function handleSending(
 ): Promise<void> {
   if (!payment.quote) throw LifecycleError.MissingQuote
 
+  // Check if the current time is greater than or equal to when the Quote should be expiring
```

Comment:
Let's add a test for this case

Reply:
Hi Max, I have been looking to add the test here at (service.test.ts) potentially something similar to ('processNext' test) or ('fund').
Is this correct?